lib/ros_value.cc

```diff
@@ -130,6 +130,80 @@ void RosValue::print(const std::string &path) const {
   std::cout << toString(path);
 }
 
+size_t RosValue::primitiveTypeToSize(const Type type) {
+  switch (type) {
+    case (Type::ros_bool):
+      return sizeof(bool);
+    case (Type::int8):
+      return sizeof(int8_t);
+    case (Type::uint8):
+      return sizeof(uint8_t);
+    case (Type::int16):
+      return sizeof(int16_t);
+    case (Type::uint16):
+      return sizeof(uint16_t);
+    case (Type::int32):
+      return sizeof(int32_t);
+    case (Type::uint32):
+      return sizeof(uint32_t);
+    case (Type::int64):
+      return sizeof(int64_t);
+    case (Type::uint64):
+      return sizeof(uint64_t);
+    case (Type::float32):
+      return sizeof(float);
+    case (Type::float64):
+      return sizeof(double);
+    case (Type::string):
+      return 0; // The size of string is unknown!
```

Comment:
It may be better to just error on this case instead, as returning 0 may get confusing.

Reply:
Will do

Reply:
Done